src/styles/blocks/main.scss

```diff
@@ -0,0 +1,90 @@
+body {
+  margin: 0;
+}
+
+.product {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 31px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  top: 1px;
+  left: 1px;
+
+  &__logo {
+    background-image: url(../images/imac.jpeg);
+    background-repeat: no-repeat;
+    background-size: 100% 99%;
+    background-position: center;
+    height: 135px;
+    width: 160px;
+    margin-bottom: 40px;
+    margin-left: 3px;
+  }
+
+  &__title {
+    @include font500;
+
+    color: $colorFont;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    @include font400;
+
+    color: $subColorFont;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__blockPrice {
+    margin-top: 23px;
+    margin-bottom: 16px;
```

Comment:
try to use either `margin-bottom` or `margin-top` to make vertical indent between elements